CONTRIBUTING.md

```diff
@@ -85,49 +85,74 @@ by post:
 
 ### Enhancement Requests and Error Reports
 
-X-Road enhancement requests and error reports can be submitted to the [X-Road Service
-Desk](https://nordic-institute.atlassian.net/servicedesk/customer/portal/4). In order 
-to access the X-Road Service Desk, [sign up](https://id.atlassian.com/signup) for an 
-account and request access to the [X-Road Backlog](https://nordic-institute.atlassian.net/jira/software/c/projects/XRDDEV/boards/6/backlog).
-Alternatively, you can use your existing account, sign in and request for access. Once 
-your request has been approved, you're able to access the [X-Road Service Desk](https://nordic-institute.atlassian.net/servicedesk/customer/portal/4) 
-and the [X-Road Backlog](https://nordic-institute.atlassian.net/jira/software/c/projects/XRDDEV/boards/6/backlog). 
-Approving the  request typically requires 1-2 business days.
+X-Road enhancement requests and error reports can be submitted directly to this GitHub repository:
+
+* For enhancement requests, please create a discussion containing your proposal in the specific softwares repository in
```

Comment:
Should be changed from "specific softwares repository" to "specific software repository" (remove the "s" from "softwares").